spec/helper/versioning_helper_spec.rb

```diff
@@ -517,4 +477,18 @@ def setup_commit_search_stubs(hashes_to_responses)
         .and_return(response)
     end
   end
+
+  def mock_commits_since_last_release(last_commit_hash, response)
+    allow(Fastlane::Actions::LastGitCommitAction).to receive(:run)
+      .and_return({
+                                                                     commit_hash: last_commit_hash
+                                                                   })
```

Comment:
indentation seems off here. Also, do we need the `()` for this one? not sure if we have a standard for this already but if feels like this would look potentially nicer:
```suggestion
        allow(Fastlane::Actions::LastGitCommitAction).to receive(:run).and_return(commit_hash: last_commit_hash)
```

Reply:
`rubocop` formatted it like this, but I like the suggestion, didn't know that was possible! I will update